docs/assessment.md

```diff
@@ -350,6 +351,581 @@ CREATE TABLE [IF NOT EXISTS] table_name
 
 [[back to top](#migration-assessment-report)]
 
+## AF300 - AF399
+The 300 series findings relate to [Compute Access mode limitations for Unity Catalog](https://docs.databricks.com/en/compute/access-mode-limitations.html#spark-api-limitations-for-unity-catalog-shared-access-mode)
+
+Resolutions may include:
+
+-  Upgrade cluter runtime to latest LTS version (e.g. 14.3 LTS)
+
+-  Migrate users with GPU, Distributed ML, Spark Context, R type workloads to Assigned clusters. Implement cluster policies and pools to even out startup time and limit upper cost boundry.
+
+-  Upgrade SQL only users (and BI tools) to SQL Warehouses (much better SQL / Warehouse experience and lower cost)
+
+-  For users with single node python ML requirements, Shared Compute with `%pip install` library support or Personal Compute with pools and compute controls may provide a better experience and better manageability.
+
+- For single node ML users on a crowded driver node of a large shared cluster, will get a better experience with Personal Compute policies combined with (warm) Compute pools
+
+### AF300.6 - 3 level namespace
+
+The `hive_metastore.` is used to refer to a 3 level namespace. Most customers will migrate hive_metastore to a workspace catalog (named based on the workspace name). The code should then map the `hive_metastore` to the appropriate catalog name.
+
+Easier solution is to define a default catalog for your session, job, cluster or workspace.
+
+Setting the Workspace default, use the admin UI or command line:
+```
+% databricks unity-catalog metastores assign --workspace-id 1234567890123456 \
+                                             --metastore-id 12345678-9999-9de3-3456-e789f0a12b34 \
+                                           --default-catalog-name my_catalog
```

Comment:
this is not used anymore, the default catalog must be set using `databricks settings update-default-workspace-namespace`

Reply:
@nkvuong Nice catch! Updated